src/EnergyPlus/PlantChillers.cc

```diff
@@ -1259,7 +1262,13 @@ namespace PlantChillers {
 					ErrorsFound = true;
 				}
 				TestCompSet( cCurrentModuleObject, cAlphaArgs( 1 ), cAlphaArgs( 13 ), cAlphaArgs( 14 ), "Heat Recovery Nodes" );
-				RegisterPlantCompDesignFlow( EngineDrivenChiller( ChillerNum ).HeatRecInletNodeNum, EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate );
+				if ( EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate == DataSizing::AutoSize ) {
+					EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRateWasAutoSized = true;
+				} else {
+					RegisterPlantCompDesignFlow( EngineDrivenChiller( ChillerNum ).HeatRecInletNodeNum, EngineDrivenChiller( ChillerNum ).DesignHeatRecVolFlowRate );
```

Comment:
But here this gets called when the chill is **not** autosized.  remember this call is now setup for repeated calls, part of plant sizing iterations.  Call it early and often, just like voting.